packages/tradershub/src/features/cfd/modals/DxtradePasswordModal/DxtradePasswordModal.tsx

```diff
@@ -148,10 +148,14 @@ const DxtradePasswordModal = () => {
 
     if (isMobile) {
         return (
-            <ModalStepWrapper renderFooter={() => renderFooter} title={' '}>
-                {successComponent}
-                {passwordComponent}
-            </ModalStepWrapper>
+            <Modal>
+                <Modal.Header title=' ' />
```

Comment:
wouldn't it be better to just make this `title` optional and assign a default value to it in the `<ModalHeader/> `component?

we only have three places in our code where we are passing empty string like this. so it would be easy to refactor them as well.
what do you think @mahdiyeh-deriv ?